src/app/pages/system/system.routing.ts

```diff
@@ -61,28 +60,19 @@ export const routes: Routes = [
         path: 'clone/:pk',
         component: BootEnvironmentCloneComponent,
         data: { title: T('Clone'), breadcrumb: T('Clone') },
-      },
-      {
-        path: 'rename/:pk',
-        component: BootEnvironmentRenameComponent,
-        data: { title: T('Rename'), breadcrumb: T('Rename') },
-      },
-      {
+      }, {
         path: 'create',
-        component: BootEnvironmentCreateComponent,
+        component: BootEnvironmentFormComponent,
         data: { title: T('Add'), breadcrumb: T('Add') },
```

Comment:
I don't think these routes are actually used. I also doubt that they would work given that `BootEnvironmentFormComponent` now expects data to be passed in from outside.